src/device/onLoad.js

```diff
@@ -300,9 +300,10 @@ realityEditor.device.onload = function () {
     }
 
     // see if we should open the modal
-    let shouldShowIntroModal = window.localStorage.getItem('neverAgainShowIntroTips') !== 'true';
+    // let shouldShowIntroModal = window.localStorage.getItem('neverAgainShowIntroTips') !== 'true';
 
-    if (shouldShowIntroModal) {
+    // if (shouldShowIntroModal) {
+    if (false) { // eslint-disable-line
```

Comment:
Make sure to revert this before merging

Reply:
I put this in on purpose, so that the Help screen does not show up. Apparently it was not a problem after all. I want to find a nicer method to show a little question mark icon once your not touching anything on the screen after you start the app. For as long as we don't have this method, the start screen is removed and therefore this if statement is just false.

Reply:
Oh, then it would be much nicer to just do `let shouldShowIntroModal = false; // old code`